Short-Answer/Algorithms_Answers.md

```diff
@@ -2,14 +2,29 @@
 
 ## Exercise I
 
-a)
+a) O(n) or linear because n is unknown. If we choose n to be 2 the function would run 8 times, however if we choose 10 it runs 1000 times.
 
 
-b)
+b) O(n log n) first part is linear second part is log n
 
 
-c)
+c) O(n) linear input not chosen yet
 
 ## Exercise II
 
+## Suppose that you have an n-story building and plenty of eggs. Suppose also that an egg gets broken if it is thrown off floor f or higher, and doesn't get broken if dropped off a floor less than floor f. Devise a strategy to determine the value of f such that the number of dropped + broken eggs is minimized.
+
+## Write out your proposed algorithm in plain English or pseudocode AND give the runtime complexity of your solution.
+
+-The base case would start off at 0 being n - story of the building.
```

Comment:
Think, do we need a base case for this solution? Base cases are utilized in recursive functions. Try thinking more generically of how you would solve this problem.